test/unit/org/apache/cassandra/utils/ASTGenerators.java

```diff
@@ -770,12 +784,12 @@ private void generateRemaining(RandomnessSource rnd,
                     }
                 }
             }
-            if (kind == Mutation.Kind.UPDATE && isTransaction)
```

Comment:
collections can do things outside of transactions, so need to push this into `supportsOperators`